src/ducks/apps/components/ApplicationPage/index.jsx

```diff
@@ -116,6 +116,7 @@ export class ApplicationPage extends Component {
               <Gallery slug={slug} images={app.screenshots} />
             )}
           <Details
+            app={app}
```

Comment:
If you pass the whole app here, maybe you can use it directly without passing the other props after? (version, source, langs, categories...)

Reply:
You're absolutely right, what do you think about doing this in another PR to limit regressions ?